docusaurus.config.js

```diff
@@ -167,242 +185,14 @@ module.exports = {
-            ],
-          },
-          {
-            to: '/docs/tools/dependabot/manage-dependabot',
-            from: [
-              '/docs/dev/versioning/manage-dependabot',
-            ],
-          },
-          {
-            to: '/docs/tools/dependabot/dependency-update-process-with-dependabot',
-            from: [
-              '/docs/dev/versioning/Dependency-Update-Process-With-Dependabot',
-            ],
-          },
-          {
-            to: '/docs/tools/dockerfile/dockerfile-use,-purpose,-and-documentation',
-            from: [
-              '/docs/dev/contributing/backend/Dockerfile-use,-purpose,-and-documentation',
-            ],
-          },
-          {
-            to: '/docs/tools/docusaurus/docusaurus',
-            from: [
-              '/docs/dev/tools/docusaurus',
-            ],
-          },
-          {
-            to: '/docs/tools/docusaurus/redocusaurus',
-            from: [
-              '/docs/dev/tools/redocusaurus',
-            ],
-          },
-          {
-            to: '/docs/tools/mockery/generate-mocks-with-mockery',
-            from: [
-              '/docs/dev/testing/writing-tests/generate-mocks-with-mockery',
-            ],
-          },
-          {
-            to: '/docs/tools/postman/setting-up-postman',
-            from: [
-              '/docs/dev/tools/Postman/Setting-Up-Postman',
-            ],
-          },
-          {
-            to: '/docs/tools/postman/creating-a-local-environment-for-postman',
-            from: [
-              '/docs/dev/tools/Postman/Creating-A-Local-Environment-for-Postman',
-            ],
-          },
-          {
-            to: '/docs/tools/postman/intercepting-cookies-for-postman',
-            from: [
-              '/docs/dev/tools/Postman/Intercepting-Cookies-for-Postman',
-            ],
-          },
-          {
-            to: '/docs/tools/postman/using-etags-and-the-if-match-header-in-postman',
-            from: [
-              '/docs/dev/tools/Postman/Using-Etags-and-the-If-Match-Header-in-Postman',
-            ],
-          },
-          {
-            to: '/docs/tools/postman/setup-postman-to-make-mutual-tls-api-calls',
-            from: [
-              '/docs/dev/tools/Postman/setup-postman-to-make-mutual-tls-api-calls',
-            ],
-          },
-          {
-            to: '/docs/tools/schemaspy/schemaspy',
-            from: [
-              '/docs/dev/tools/SchemaSpy',
-            ],
-          },
-          {
-            to: '/docs/tools/telemetry/running-telemetry-locally',
-            from: [
-              '/docs/dev/tools/telemetry/Running-telemetry-locally',
-            ],
-          },
-        ],
+        redirects: (() => {
+          let redirects = [];
+          redirects.push(FrontendPages);
+          redirects.push(BackendPages);
+          redirects.push(ToolsPages);
+          redirects.push(APIPages);
+          return redirects.flat();
+        })(),
```

Comment:
To me this makes a lot more sense given how large this file is becoming. There is probably more things we could modularize under the `utils/` directory pattern.